near/kycdao-ntnft/src/lib.rs

```diff
@@ -107,12 +175,17 @@ impl KycdaoNTNFT {
         let authorized_opt = self.authorized_token_metadata.get(&digest);
         assert!(authorized_opt.is_none(), "Code already authorized");
 
+        let new_status = status.unwrap_or_default();
+
         self.authorized_token_metadata.insert(&digest, &metadata);
+        self.authorized_statuses.insert(&digest, &new_status);
     }
 
     /*****************
     Public interfaces
     *****************/
+    pub fn version(&self) -> &str { "0.2.0" }
```

Comment:
Nice - good to have the same version info on the contract, as this can be upgraded as well right. Have you started a CHANGELOG.md to track changes?

Reply:
Not yet, I'm debating if we should move the changelog to the root and make it "global" so the versions on different chains can be linked, or if we should have separate changelogs for every language?

Reply:
Ah, that's a good point. My only worry is that the CHANGELOG gets a bit noisy... but I think all the changes are generally grouped into versions. It means there's a tendency for us to consider each change we make to a contract in the context of how it affects the others more naturally, and I think that's a good thing.

Yea - I think moving to a root level CHANGELOG makes sense. There's still unresolved PRs with CHANGELOG changes though, I'd say add CHANGELOG details to this PR separately and we can combine after all PRs are merged.